tests/Functional/App/config/cascadeOnScalars/mapping/Mutation.types.yml

```diff
@@ -4,7 +4,7 @@ Mutation:
         fields:
             cascadeOnScalar:
                 type: Boolean
-                resolve: '@=mut("mutation_mock", [args, validator])'
+                resolve: '@=mut("mutation_mock", args, validator)'
```

Comment:
How this could work since the change of the signature? 🤔

Reply:
Forgot to change the alias from `mut` to `m` in the service definition.

Reply:
Fixed here https://github.com/overblog/GraphQLBundle/pull/817